WalletWasabi/WebClients/ChaumianCoinJoin/WasabiClient.cs

```diff
@@ -24,28 +24,26 @@ public WasabiClient(Uri baseUri, IPEndPoint torSocks5EndPoint = null) : base(bas
 
 		#region blockchain
 
-		public async Task<IEnumerable<FilterModel>> GetFiltersAsync(uint256 bestKnownBlockHash, Height blockHeight, int count)
+		public async Task<IEnumerable<string>> GetFiltersAsync(uint256 bestKnownBlockHash, int count)
 		{
 			using(var response = await TorClient.SendAndRetryAsync(HttpMethod.Get,
-																	HttpStatusCode.OK, 
-			                                                       $"/api/v1/btc/blockchain/filters?bestKnownBlockHash={bestKnownBlockHash}&count={count}"))
+																	HttpStatusCode.OK,
+																	$"/api/v1/btc/blockchain/filters?bestKnownBlockHash={bestKnownBlockHash}&count={count}"))
 			{
 				if (response.StatusCode == HttpStatusCode.NoContent)
 				{
-					return Enumerable.Empty<FilterModel>();
+					return Enumerable.Empty<string>();
 				}
 				if (response.StatusCode != HttpStatusCode.OK)
 				{
 					string error = await response.Content.ReadAsJsonAsync<string>();
 					var errorMessage = error == null ? string.Empty : $"\n{error}";
-					throw new HttpRequestException($"{response.StatusCode.ToReasonString()}{errorMessage}");
+					throw new HttpRequestException($"{response.StatusCode.ToReasonString()}: {errorMessage}");
```

Comment:
If he `errorMessage` is empty, then it will look something like "Bad Request:".

Reply:
If you look at the `errorMessage`, then you can see if it's not empty, then a `\n` is inserted, so please remove this `:`.